BibBuddy/app/src/main/java/de/bibbuddy/BookFragment.java

```diff
@@ -60,6 +61,39 @@
 
   private ExportBibTex exportBibTex;
   private ImportBibTex importBibTex;
+  ActivityResultLauncher<Intent> filePickerActivityResultLauncher = registerForActivityResult(
```

Comment:
Please add a newline.
```suggestion

  ActivityResultLauncher<Intent> filePickerActivityResultLauncher = registerForActivityResult(
```